test/viz/test_widget_list.py

```diff
@@ -102,20 +79,22 @@ def test_widget_list_get_widgets_info(self):
         self.assertEqual(widgets_info, [
```

Comment:
We may include an animation widget in this test to cover default prop and custom prop.